devices/collect/tim/ccr.yaml

```diff
@@ -20,14 +28,16 @@ CCER:
       description: Capture/Compare %s output Polarity
     "?~CC?NE":
       description: Capture/Compare %s complementary output enable
-"CCMR[12]_Input":
+"CCMR?_Input":
   _array:
-    CC?S:
-      description: Capture/Compare %s selection
     IC?PSC:
       description: Input capture %s prescaler
     IC?F:
       description: Input capture %s filter
+"?~CCMR2_Input*":
+  _derive:
+    IC%sF: CCMR1_Input.IC%sF
+    IC%sPSC: CCMR1_Input.IC%sPSC
```

Comment:
What does the `?~` mean here?

Also, will the `%s` be replaced by whatever the `*` matched?

Reply:
`?~` means skip rule if name does not exists.
Without this it will cause "not found error".

`%s` is array suffix placeholder. See SVD specification. https://arm-software.github.io/CMSIS_5/SVD/html/elem_special.html#dimElementGroup_gr

Reply:
Ok, thanks